central/notifiers/email/email.go

```diff
@@ -392,8 +395,10 @@ func (e *email) send(ctx context.Context, m *message) error {
 		}
 	}
 
-	if err = client.Auth(auth); err != nil {
-		return createError("SMTP authentication failed", err)
+	if !e.notifier.GetEmail().GetAllowUnauthenticatedSmtp() {
```

Comment:
I'm not sure if the existing unit tests are set up for it, but it would be good to get a unit test for this

Reply:
Added overall test unauth email in the existing integration test